WordPress/src/main/java/org/wordpress/android/support/ZendeskHelper.kt

```diff
@@ -462,6 +464,7 @@ private val wpcomPushNotificationDeviceToken: String?
 
 private object ZendeskConstants {
     const val articleLabel = "Android"
+    const val androidTag = "Android"
```

Comment:
I wondered about a new const vs just using `articleLabel` but I think it gives some extra context so 👍 
Now my paranoia is kicking in and I'm imagining a scenario where someone later sees these consts have the same value and removes one in favor of the other.  Then another someone edits the value cos its no longer clear how its used.  (Super paranoid right?)
Would it make sense to add a comment here clarifying our Zendesk filters rely on the value and it shouldn't be changed?

Reply:
That's a little bit too much paranoia for my taste 😱 Let me think how to avoid this impossible case! 😁

Reply:
😆

Reply:
I renamed the value to `platformTag` hoping that makes the usage more obvious. I also added a comment specifically saying it should be kept separate from the `articleLabel`. Let me know what you think!